### PR TITLE
Localisation readme

### PR DESCRIPTION
**Pull request by kavi-egov**

### CODE REVIEW

File: egov-localization/LOCALSETUP.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+- [X] Redis
+- [ ] Elasticsearch
+- [ ] Kafka
+  - [ ] Consumer
+  - [ ] Producer
+
+## Running Locally
+
+To run this services locally, you need to port forward below services locally
+
```
Comment (tarunlalwani) [inline egov-localization/LOCALSETUP.md:19]: If there are no services at least remove the section

File: egov-localization/README.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+
+
+### Service Dependencies
+
+
+
+### Swagger API Contract
+
+
```
Comment (tarunlalwani) [inline egov-localization/README.md:17]: API contract?

**Review by tarunlalwani — CHANGES_REQUESTED** [review]

**Comment by kavi-egov** [comment]
Api contract is not available. We have create for some of the core services.

File: egov-localization/README.md (hunk 2 above)
Comment (kavi-egov) [reply egov-localization/README.md:17]: API contract not available. we have to write new

**Review by tarunlalwani — APPROVED** [review]

**Pull request merged**